### ISSUE TITLE
Phone frame for site screenshots

### ISSUE THREAD

**Issue by cheeaun** [tastejs/hacker-news-pwas#14]
Continuing the discussion from https://github.com/tastejs/hacker-news-pwas/issues/7#issuecomment-297613778

The question: 

> Just wondering, why are the phone mockups (Samsung phone, HTC phone) included as part of the site screenshots? I think would be better if just use a generic phone outline vector?

Additional concerns:

- Each site/app may seem *affiliated* to the phone brand/model?
- The screenshot images should be separated from the phone frame itself (in case we change to other phone frames or remove them later)

Possible solution(s):

- Change the phone frame to a generic one, or just an outline?

**Comment by housseindjirdeh** [comment]
Thanks @cheeaun, yeah there was no particular reason why I decided to just random brand/models. Used [MockUPhone](https://mockuphone.com) hence why the screenshot images aren't separated from the phone frame.

We can use a single one from the same site (like a [Nexus](https://mockuphone.com/nexus5x)) or use an outline around each screenshot. I think either solution should be okay.

**Comment by addyosmani** [comment]
Yeah I think those are valid concerns. It also makes it easier to update screenshots in the future as/when apps change if they aren't tied to the phone frame.

> We can use a single one from the same site (like a Nexus) or use an outline around each screenshot. I think either solution should be okay.

WFM

I haven't used it before, but there's also a CSS-only version of some of the frames available here https://marvelapp.github.io/devices.css/ if we wanted to go down that path (7KB).
